firmware/robot2015/src-ctrl/main.cpp

```diff
@@ -245,6 +245,14 @@ int main() {
             errorBitmask |= !status.hallOK << pair.second;
         }
 
+        // get the battery voltage
+        AnalogIn batt(RJ_BATT_SENSE);
+
+        uint8_t batt_vol = (batt.read_u16() >> 8);
+
+        const float batt_vol_in_soccer = batt_vol * 0.09884f;
```

Comment:
It'd be good to make this a constant in `rtp.hpp` so the magic number only exists in one place.